sdk/gramine/0001-SGX-Step-Gramine-patches-to-reconfigure-AEP-TCS.patch

```diff
@@ -1,22 +1,27 @@
-From 01802a7641e0090e3a4926239ca004d7de2e6a5c Mon Sep 17 00:00:00 2001
-From: Jo Van Bulck <jo.vanbulck@cs.kuleuven.be>
-Date: Wed, 12 Oct 2022 19:14:24 +0200
-Subject: [PATCH 1/2] SGX-Step Gramine patches to reconfigure AEP/TCS.
-
----
- pal/src/host/linux-sgx/host_entry.S     | 52 +++++++++++++++++++++++--
- pal/src/host/linux-sgx/host_exception.c | 13 +++++++
- pal/src/host/linux-sgx/meson.build      | 11 ++++++
- 3 files changed, 73 insertions(+), 3 deletions(-)
-
+diff --git a/meson.build b/meson.build
+index 85dfcc6b..faae77cc 100644
+--- a/meson.build
++++ b/meson.build
+@@ -272,6 +272,10 @@ if sgx
+         sgx_dcap_quoteverify_dep = cc.find_library('sgx_dcap_quoteverify')
+     endif
+ 
++  # XXX taken from: https://stackoverflow.com/questions/34663124/link-static-library-to-shared-library-or-to-a-binary
++  libsgxstepdir = '<ENTER ABSOLUTE PATH OF LIBSGXSTEP HERE>'
```

Comment:
not sure, why is the manual absolute path needed here?

For me, I could compile and link gramine and libsgxstep with my earlier [patch](https://github.com/jovanbulck/sgx-step/blob/master/sdk/gramine/0001-SGX-Step-Gramine-patches-to-reconfigure-AEP-TCS.patch).

If we can avoid this manual entering of the absolute path twice here, that would certainly be easier :)

Reply:
If I remember correctly it would compile but then have problems finding the functions from the library later on (it's been a while though)